content/md/en/docs/reference/command-line-tools/srtool.md

```diff
@@ -1,9 +1,409 @@
 ---
 title: srtool
-description: Command-line reference information for the srtool program.
+description: Describes the Docker container and command-line interface for building the Substrate WebAssembly runtime.
 keywords:
 ---
 
-The `srtool` allows building WASM runtimes in a deterministic way, allowing CIs and users to produce a strictly identical WASM runtime.
+The Substrate runtime tool (`srtool`) is a Docker container that enables you to build the Substrate WebAssembly runtime in a deterministic way.
```

Comment:
It may be worth mentioning that `srtool` started as a docker image but grew as a set of tools around the docker image.

Reply:
good suggestion.